training_scripts/gsgnn_mt/README.md

```diff
@@ -0,0 +1,30 @@
+# Multi-task Learning Example Yaml Files
+This folder presents example yaml files for multi-task learning with Movielens datasets.
+
+## Build a graph for multi-task learning on Movielens dataset
+```
+python3 $GS_HOME/tests/end2end-tests/data_gen/process_movielens.py
+
+python3 -m graphstorm.gconstruct.construct_graph \
+	--conf-file $GS_HOME/tests/end2end-tests/data_gen/movielens_multi_task.json \
+	--num-processes 1 \
+	--output-dir movielen_100k_multi_task_train_val_1p_4t \
+	--graph-name movie-lens-100k \
+	--add-reverse-edges
+```
+
+## Run the example
+```
+python3 -m graphstorm.run.gs_multi_task_learning \
+	--workspace $GS_HOME/training_scripts/gsgnn_mt  \
+	--num-trainers 4 \
```

Comment:
Because here is 1 partition, we can change or remove some unnecessary arguments, include:
`--num-trainer 1`, use 1 works for both CPUs and one GPU instance.
Remove:
`--num-samplers 0`
`--ip-config ip_list.txt`
`--ssh-port 2222`
`--logging-file /tmp/train_log.txt`
`--logging-level debug`